service/history/queue/interface.go

```diff
@@ -56,6 +56,7 @@ type (
 		Split(ProcessingQueueSplitPolicy) []ProcessingQueue
 		Merge(ProcessingQueue) []ProcessingQueue
 		AddTasks(map[task.Key]task.Task, task.Key)
+		GetTasks() map[task.Key]task.Task
```

Comment:
Is it possible to use a different name, like `GetPendingTasks()` or `GetOutstandingTasks()`? I think in the future we will moving task loading logic into processing queue and the `GetTask` name should be reserved for loading new tasks from persistence or the in memory cache.